src/components/PageTitle.jsx

```diff
@@ -3,19 +3,19 @@ import wave from "@/public/waveBottom.webp";
 
 const PageTitle = ({ image, title }) => {
   return (
-    <div className="w-full relative">
+    <div className="w-full relative mt-12 md:mt-0 ">
       <Image
         src={image}
         alt="background image"
-        className="w-screen opacity-80"
+        className="w-screen h-40 md:h-auto opacity-80"
       />
-      <p className="flex items-center justify-center absolute top-0 font-bold text-5xl text-white border-swim-yellow border-r-[20px] w-1/3 h-full bg-swim-blue-200/70">
+      <p className="flex items-center justify-center absolute top-0 font-bold text-2xl md:text-5xl text-white border-swim-yellow border-r-[20px] w-1/3 h-full bg-swim-blue-200/70">
         {title}
       </p>
       <Image
         src={wave}
         alt="wave bottom image"
-        className="absolute bottom-0 transform -scale-x-100 w-screen"
+        className="absolute bottom-0 transform -scale-x-100 w-screen md:w-auto "
```

Comment:
![image](https://github.com/acm-ucr/ucr-swim-website/assets/91030482/a53a327f-59df-44d3-8ba3-d0371790e4f4)

the wave is broken on desktop view